spacy_llm/tasks/templates/ner.v3.jinja

```diff
@@ -0,0 +1,64 @@
+You are an expert Named Entity Recognition (NER) system.
+Your task is to accept Text as input and extract named entities.
+{%- if allow_overlap -%}
+{# whitespace #}
+The entities you extract can overlap with each other.
```

Comment:
Wait, should we have this in the NER template? Shouldn't we ask for non-overlapping spans, and only allow overlap for the spancat template?

Reply:
It was originally one template for both SpanCat and NER. For consistency reasons I split it in two and named them after the task versions. 
Both templates are fixed in https://github.com/explosion/spacy-llm/pull/180/commits/c763689fbd95c02a7d64b13d33a45a89c1edc14e.